admin/src/pages/EditView/form-schema.js

```diff
@@ -7,17 +7,18 @@ import {
   URL_MAILTO_REGEX,
   URL_TEL_REGEX,
 } from '../../constants';
+import { getTrad } from '../../utils';
 
 const itemSchema = yup.object().shape( {
   order: yup
     .number( translatedErrors.number )
-    .required(),
+    .required( translatedErrors.required ),
   title: yup
     .string( translatedErrors.string )
-    .required(),
+    .required( translatedErrors.required ),
   url: yup
     .string( translatedErrors.string )
-    .test( value => !! (
+    .test( 'is-url', getTrad( 'error.url.invalid' ), value => !! (
```

Comment:
@davidlueder Just for my understanding, what is the benefit of the first 2 arguments in `test()`?